src/MOI/MOI_wrapper.jl

```diff
@@ -132,6 +132,26 @@ mutable struct ConstraintInfo
     ConstraintInfo(row::Int, set::MOI.AbstractSet, type::ConstraintType) = new(row, set, "", type)
 end
 
+mutable struct NLPVariableInfo
+    lower_bound::Union{Float64, Nothing}
+    upper_bound::Union{Float64, Nothing}
+    category::Symbol
+    start::Union{Float64, Nothing}
+    name::Union{String, Nothing}
+end
+NLPVariableInfo() = NLPVariableInfo(nothing, nothing, :Cont, nothing, nothing)
```

Comment:
All this info already exists elsewhere. this should be removed as it is not necessary.